webpack.config.js

```diff
@@ -154,13 +154,21 @@ if ( isDevelopment || isDesktop ) {
 const cssFilename = cssNameFromFilename( outputFilename );
 const cssChunkFilename = cssNameFromFilename( outputChunkFilename );
 
-const fileLoader = FileConfig.loader( {
-	// File-loader does not understand absolute paths so __dirname won't work.
-	// Build off `output.path` for a result like `/…/public/evergreen/../images/`.
-	outputPath: path.join( '..', 'images' ),
-	publicPath: '/calypso/images/',
-	emitFile: browserslistEnv === defaultBrowserslistEnv, // Only output files once.
```

Comment:
Was it intentional to remove the `emitFile` option? It prevents writing the same files twice when doing parallel evergreen/fallback builds.

Reply:
I did remove it intentionally because I had some concerns about the fragility of writing it once and hoping that the default always runs. I'm not sure how expensive these redundant writes are.

We can restore it if you prefer.

Reply:
I'm a bit worried about the files getting corrupted when two processes at once try to write them. No so much about performance really. I'd prefer to restore them. And always emitting in development, i.e., leaving the `emitFile` option undefined there.

Reply:
👍 Restored in e2c7c6620d3e5f6ea04a3c1e134f8e1d4450216f